src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -1,5 +1,6 @@
 #include "library/rekordbox/rekordboxfeature.h"
 
+#include <mixer/playerinfo.h>
```

Comment:
this is a local include. we use "" for that.

```suggestion
#include "mixer/playerinfo.h"
```